### PR TITLE
Fix docker based localhost setup documentation.

### PR DESCRIPTION
**Pull request by vishh**

### CODE REVIEW

File: docs/getting-started-guides/docker.md [hunk 1]
```diff
@@ -53,39 +53,44 @@ At this point you should have a running Kubernetes cluster.  You can test this
...
+kubectl config use-context test-doc
 ```
 
-For Max OS X users instead of `localhost` you will have to use IP address of your docker machine,
-which you can find by running `docker-machine env <machinename>` (see [documentation](https://docs.docker.com/machine/reference/env/)
-for details).
+On OS X, to make the API server accessible locally, setup a ssh tunnel.
+
+```shell
+docker-machine ssh $DOCKER_MACHINE_NAME -N -L 8080:localhost:8080
```
Comment (pwittrock) [inline docs/getting-started-guides/docker.md]: plz add `docker-machine ls` so the user knows how to get the name

Comment (vishh) [inline docs/getting-started-guides/docker.md]: Fixed the basic issue in a different way. PTAL

**Comment by vishh** [comment]
PTAL @pwittrock @johndmulhausen

**Comment by pwittrock** [comment]
Very nice.  Looks good to me.  Many thanks!

File: docs/getting-started-guides/docker.md [hunk 2]
```diff
@@ -53,39 +53,44 @@ At this point you should have a running Kubernetes cluster.  You can test this
...
 ```shell
-$ wget http://storage.googleapis.com/kubernetes-release/release/v${K8S_VERSION}/bin/linux/amd64/kubectl
-$ chmod 755 kubectl
-$ PATH=$PATH:`pwd`
+wget http://storage.googleapis.com/kubernetes-release/release/v${K8S_VERSION}/bin/linux/amd64/kubectl
+chmod 755 kubectl
+PATH=$PATH:`pwd`
 ```
 
 Create configuration:
```
Comment (luxas) [inline docs/getting-started-guides/docker.md]: nit: I see this as optional. 
It's not required for beginners to create a config when they only want to start hacking.
Please `Create configuration (optional):`

**Comment by luxas** [comment]
Haha, you were a bit faster than me :)
Looks good to me after the nit.

**Comment by johndmulhausen** [comment]
Thanks @pwittrock for the tech review, and thanks to @vishh for the PR. :)

**Pull request merged**